src/sass/components/RefileWarning.scss

```diff
@@ -9,7 +9,8 @@
 
 .RefileWarning {
   animation: 0.3s ease-out 0s 1 animateWarningHeader;
-  margin: 1px 0 0 0;
+  margin: -1px 0 0 0;
+  box-shadow: 0px 1px 3px -2px;
```

Comment:
I don't think this is necessary. The background color here provides a border between the content and the warnings.